### PR TITLE
CI: Update `grafana/build-container` version

### PR DESCRIPTION
**Pull request by dsotirakis**
**What this PR does / why we need it**:

Updates `grafana/build-container` version to `1.4.4`, which includes node and yarn updates.

### CODE REVIEW

File: scripts/build/ci-build/Dockerfile [hunk 1]
```diff
@@ -113,8 +113,8 @@ FROM debian:stretch-20210208
 ENV GOVERSION=1.17 \
     PATH=/usr/local/go/bin:$PATH \
     GOPATH=/go \
-    NODEVERSION=14.17.6-1nodesource1 \
-    YARNVERSION=3.1.0-rc.5
+    NODEVERSION=14.18.1-1nodesource1 \
+    YARNVERSION=1.22.15-1
```
Comment (kaydelaney) [inline scripts/build/ci-build/Dockerfile:117]: Should this be
```suggestion
    YARNVERSION=3.1.0
```
?

Comment (dsotirakis) [reply scripts/build/ci-build/Dockerfile:117]: Thought so too, but I get the error that this version doesn't exist.
Actually, if you don't specify a version, the latest version it gets is this one. (`1.22.15-1`)

Comment (dsotirakis) [reply scripts/build/ci-build/Dockerfile:117]: If this Drone run passes, then I think we are okay to go :)

File: scripts/build/ci-build/build-deploy.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eo pipefail
 
-_version="1.4.3"
+_version="1.4.5"
```
Comment (kaydelaney) [inline scripts/build/ci-build/build-deploy.sh:4]: What happened to `1.4.4`? 🤔

Comment (dsotirakis) [reply scripts/build/ci-build/build-deploy.sh:4]: 1.4.4 is on this PR.
Next time we want to update the image, we run the build-deploy.sh and the new version 1.4.5 will be deployed :) 
We just forgot to update the script for 1.4.4 last time!

However, if you too find it annoying, I can remove the script, as its easy to do it manually!

Comment (kaydelaney) [reply scripts/build/ci-build/build-deploy.sh:4]: Aha, I get it 😄

**Review by kaydelaney — APPROVED** [review]

**Pull request merged**